easybuild/easyconfigs/b/BayesTraits/BayesTraits-4.1.2-Linux.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = "Tarball"
```

Comment:
@asalois I know you've changed it a bit, but please restore the attribution from the previous version.
You're welcome to add your own attribution below it.